cmd/brew-brewdle.rb

```diff
@@ -1,5 +1,37 @@
 #!/usr/bin/env ruby
 
 BREWDLER_ROOT = File.expand_path "#{File.dirname(__FILE__)}/.."
-ENV["RUBYLIB"] = "#{BREWDLER_ROOT}/lib:#{ENV["RUBYLIB"]}"
-exec "#{BREWDLER_ROOT}/bin/brewdle"
+BREWDLER_LIB = Pathname.new(BREWDLER_ROOT)/"lib"
+
+$LOAD_PATH.unshift(BREWDLER_LIB)
+
+require "brewdler"
+
+usage = <<-EOS.undent
+  brew brewdle
+  brew brewdle [--version]
+  brew brewdle [-h|--help]
+
+  Usage:
+  Bundler for non-Ruby dependencies from Homebrew
+
+  Options:
+  -h, --help        show this help message and exit
+  --version         show the version of brewdler
+EOS
+
+if ARGV.flag?("--version")
```

Comment:
Homebrew (like Ruby) uses `-v` for verbose when other arguments are specified and for the version as a single argument.